soundzones_protocol/documentation_SZP.md

```diff
@@ -0,0 +1,257 @@
+
+# Documentation for SoundZone Protocol
+<!-- 
+To compile puml use: (Assuming plantuml you are in the directory)
+plantuml.jar -tsvg readme.md -o sequence_diagrams
+-->
+
+version = 0.0.2
+
+This file will document SZP.
+
+As it is a 7th layer protocol, it is treated as a Point-Point communication
+
+| Defines | Values |
+|---|---|
+| IPv | IPv4 |
+| Transport protocol | UDP |
+| Port | 1695 ( Seemingly not used ) |
+| Transmission encoding | utf-8 |
+
+## Main Structure
+
+[CID, {payload} ]
+
+* CID - Command Id
+* payload - Data to send (Structure depends on the CID)
```

Comment:
Er ikke helt med på din structur, skal der sendes en ny pakke for hver command. Kan der ikke sendes flere commands i en pakke?